dev-libs/gobject-introspection/gobject-introspection-1.76.1.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{10..12} )
```

Comment:
Don't make changes to other ebuilds in version bumps (except -9999 ebuilds to keep them in sync).